README.md

```diff
@@ -1,37 +1,61 @@
 # NEAR Protocol Specifications and Standards
 
 This repository hosts the current NEAR Protocol specification and standards.
-Including core protocol specification, APIs, contract standards, processes and workflows.
+This includes the core protocol specification, APIs, contract standards, processes, and workflows.
 
-Both changes to the core specification and standards are called Near Enhancement Proposals (NEPs).
+Changes to the core specification and standards are called NEAR Enhancement Proposals (NEPs).
+
+This repository uses [mdBook](https://rust-lang.github.io/mdBook) for the [Nomicon website](https://nomicon.io).
 
 ## Specification
 
-NEAR Specification is under active development. Latest version can be found - https://nomicon.io
-Specification defines how any NEAR client should be connecting, producing blocks, reaching consensus, processing state transitions and runtime APIs.
+NEAR Specification is under active development.
+Specification defines how any NEAR client should be connecting, producing blocks, reaching consensus, processing state transitions, using runtime APIs, and implementing smart contract standards as well.
```

Comment:
I don't think implementing smart contract standards is part of the client. We may want to have a separate sentence for this.